tfhe/src/shortint/engine/mod.rs

```diff
@@ -239,18 +242,25 @@ impl ShortintEngine {
     fn generate_accumulator_bivariate_with_engine<F>(
         server_key: &ServerKey,
         f: F,
-    ) -> EngineResult<Accumulator>
+        factor: MessageModulus,
```

Comment:
is there a way to have a better name for this ? I'm guessing it's linked to the various usage in the code base

Reply:
Yeah i'm not a huge fan of this name but I don't have another proposition for now

Reply:
left_message_scaling ? as it's effectively by how much we scale the left message to put it in the MSB/carry part of the ciphertext ?